pkg/flare/archive.go

```diff
@@ -113,6 +113,16 @@ func CreatePerformanceProfile(prefix, debugURL string, cpusec int, target *Profi
 			Name: prefix + "-2nd-heap.pprof",
 			URL:  debugURL + "/heap",
 		},
+		{
+			// mutex profile
+			Name: prefix + "-mutex.pprof",
+			URL:  debugURL + "/mutex",
```

Comment:
Where this endpoint is defined?

Reply:
It is provided by `pprof` package: https://golang.org/pkg/net/http/pprof/